tests/integration/modifiers/sortable-group-test.js

```diff
@@ -54,6 +54,49 @@ module('Integration | Modifier | sortable-group', function(hooks) {
     assert.equal(contents('#test-list'), 'Tres Dos Uno');
   });
 
+  test('Announcer has appropriate text for user actions', async function (assert) {
+    this.items = ['Uno', 'Dos', 'Tres'];
+
+    this.update = (items) => {
+      set(this, 'items', items);
+    };
+
+    await render(hbs`
+      <ol id="test-list" {{sortable-group onChange=this.update}}>
+        {{#each this.items as |item|}}
+          <li {{sortable-item model=item}} >
+            {{item}}
+            <button data-test-handle={{item}} {{sortable-handle}}>
+              handle
+            </button>
+          </li>
+        {{/each}}
+      </ol>
+    `);
+
+
+    triggerKeyEvent('[data-test-handle=Uno]', 'keydown', 32) /* SPACE */;
+
+    await announcerHasText();
+    assert.dom(announcerSelector).hasText('item at position, 1 of 3, is activated to be repositioned.Press up and down keys to change position, Space to confirm new position, Escape to cancel.');
+
+    triggerKeyEvent('[data-test-handle=Uno]', 'keydown', 40) /* DOWN */;
```

Comment:
Should we await `triggerKeyEvent`? Does`waitUntil` from `await announcerHasText` properly `await` for settleness?

Reply:
I can't await settledness, because that would be too late to catch the announcement text. The announcement text gets removed from the dom pretty quickly. The `await announcerHasText` basically polls the dom for changes to the announcer text element and as soon as it changes it resolves.

Reply:
gotcha.